src/components/Related/Taxonomy/Sunburst/style.css

```diff
@@ -0,0 +1,33 @@
+.sunburst {
+  & h5 {
+    font-weight: normal;
+    margin: 0.5em 0;
+  }
+
+  & .legends {
+    list-style: none;
+    margin: 0;
+  }
+  & dl {
+    & dt {
+      display: block;
+      font-weight: bold;
+      margin-bottom: 0;
+    }
+    & dd {
+      display: block;
+      font-size: 0.85em;
+      padding-bottom: 0.2rem;
+    }
+  }
+  & .sunburst-depth {
+    display: flex;
+    align-items: center;
+    word-break: keep-all;
+
+    & input {
+      margin-left: 0.7em;
+      margin-right: 0.7em;
+    }
+  }
+}
```

Comment:
The extension is `.css`, but it looks like SCSS. Any reason for not name the file `style.scss`?

Reply:
It's because we are not using sass there, it is actually postcss, and it gets process in webpack together with normal css.

Reply:
I see. Thank you for the clarification. :)